### PR TITLE
clarify cashout

### PR DESCRIPTION
**Pull request by traumschule**

### CODE REVIEW

File: src/locales/en/token.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
       },
       "howToCashout": {
         "title": "How do I cashout tJOY to USD?",
-        "text": "We use Bitcoin Cash (BCH) as a payment rail on our testnets. You can cashout tJOY by making a transfer to the dedicated burn address: <0>5D5PhZQNJzcJXVBxwJxZcsutjKPqUPydrvpu6HeiBfMaeKQu</0> Before doing this, you should set the <0>memo</0> of the key from which the transfer is made to your BCH address. At the moment of the transfer, the BCH to USD exchange rate is locked. <1/><1/>More information on payouts can be found <2>here</2>."
+        "text": "We use Bitcoin Cash (BCH) as a payment rail on our testnets. You can cashout tJOY by making a transfer to the dedicated burn address: <0>5D5PhZQNJzcJXVBxwJxZcsutjKPqUPydrvpu6HeiBfMaeKQu</0> Before doing this, you should set the <0>memo</0> to your BCH address for the key from which the transfer is made from. The actual BCH to USD exchange rate is used we send the BCH. <1/><1/>More information on payouts can be found <2>here</2>."
```
Comment (blrhc) [inline src/locales/en/token.json:41]:
```suggestion
        "text": "We use Bitcoin Cash (BCH) as a payment rail on our testnets. You can cashout tJOY by making a transfer to the dedicated burn address: <0>5D5PhZQNJzcJXVBxwJxZcsutjKPqUPydrvpu6HeiBfMaeKQu</0> Before doing this, you should set the <0>memo</0> of the key which the transfer is made from to your BCH address. The USD to BCH exchange rate calculated at the moment of sending the BCH. <1/><1/>More information on payouts can be found <2>here</2>."
```

**Review by blrhc — CHANGES_REQUESTED** [review]
Does this look OK?
@traumschule

**Comment by traumschule** [comment]
@blrhc Sure!

**Comment by DzhideX** [comment]
Right now we're actually finishing up the new automated Cashout system and this part will have to be rewritten anyways so I think it's best if we update it as part of the [Cashout UI Implementation PR](https://github.com/Joystream/joystream-org/pull/436).

**Pull request closed without merging**